File: FriendMusic/FriendMusic/Models/Song.cs
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace FriendMusic.Models
{
    public class Song
    {
        public int Id { get; set; }

        [Display(Name = "Song Name")]
        //[StringLength(50)]
        public string Title { get; set; } = String.Empty;

        [Required]
        //[Display(Name = "Sound Specialist Esquire The Third")]
        public string Artist { get; set; } = String.Empty;
        
        [JsonConverter(typeof(JsonTimeSpanConverter))]
        public TimeSpan Length { get; set; }
        public string AlbumTitle { get; set; } = String.Empty;

        public List<Playlist> Playlists { get; set; } = new List<Playlist>();
    }
}

File: FriendMusicWeb/friend-music-web/src/models/song.ts
export interface Song {
    Id: number;
    Title: string;
    Artist: string;
    Length: string;
    AlbumTitle: string; 
}

export interface SongToAdd {
    Title: string;
    Artist: string;
    Length: string;
    AlbumTitle: string; 
}

export interface SongToUpdate {
    Id: number;
    Title: string;
    Artist: string;
    Length: string;
    AlbumTitle: string; 
}

export default Song;